picfixPy/vibrance.py

```diff
@@ -1,20 +1,54 @@
-def vibrance(input_img, intensity=5, output_img):
-    
+def vibrance(input_img, intensity=0, display = False, output = ''):
+
     '''
-    Adjust the vibrance of an image, given the intensity specificication
+    Increase or decrease the vibrance of an image, given the intensity specificication.
 
     Input
     ============
-    input_img -- string of path for an image file in .png format
-    intensity -- integer in range [0,10] to specificy intensity of vibrance enhancement applied to the image,
-                 defaults to 5.
-    output_img -- string path for output .png image with adjusted vibrance
+
+    img -- string. Path for an image file in .jpg, .jpeg, .png, .tiff format.
+    intensity -- integer/float in range [-10,10]. Specificies intensity of vibrance applied to image.
```

Comment:
Interesting choice for supporting negative intensity values again, I don't think I'll do this for contrast though, because adding negative intensity mean writing more test cases :) I'm fine with having negative intensity values for the vibrance function as long as you can set up the appropriate test cases for it.
```suggestion
    intensity -- integer/float in range [-10,10]. Specifies intensity of vibrance applied to image. Defaults to 5.
```

Reply:
I got the greyscale test ready to go!